README.md

```diff
@@ -490,6 +490,47 @@ The available sort tokens are:
 - name
 - path
 
+##### Restrict results and hierarchical values to non-root level
+
+By default the hierarchical facet will need the root level
```

Comment:
I feel the documentation is not easy to understand here.

Maybe we can do an example based on your usecase:

```
let's say you have a lot of levels:
- fruits
  - yellow
    - citrus
      - spicy

But you only want to get the values starting at "citrus", you could do: prefixPath..
```

Something like that?